frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceCardModal.tsx

```diff
@@ -201,7 +202,7 @@ export default _.compose(
   }),
   Collections.load({
     id: (state: State, { card }: ModalCardProps) =>
-      card?.collection_id ?? "root",
+      card ? coerceCollectionId(card.collection_id) : undefined,
```

Comment:
If we have card, we can load its collection. When we don't - don't try to load the root collection because it can fail.